eng/Versions.props

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This file has been deprecated in WindowsAppSDk related repositories. Please Use Version.Dependencies.props -->
```

Comment:
NIT: "See Version.Dependencies.props, Version.Dependencies.xml and Version.Details.xml"